### PR TITLE
Insurrection

### PR DESCRIPTION
**Pull request by Geevies**
A gamemode which features Revolutionaries and Borers.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/gamemodes/mixed/uprising.dm [hunk 1]
```diff
@@ -5,7 +5,5 @@
 	extended_round_description = "Traitors and revolutionaries spawn in this round."
 	required_players = 15
 	required_enemies = 3
-	end_on_antag_death = 1
 	antag_tags = list(MODE_REVOLUTIONARY, MODE_LOYALIST, MODE_TRAITOR)
-	require_all_templates = 1
-	votable = 1
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/mixed/uprising.dm]: You forgot to readd this.

Comment (Geevies) [reply code/game/gamemodes/mixed/uprising.dm]: Not needed.

File: code/game/gamemodes/mixed/insurrection.dm [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/datum/game_mode/insurrection
+	name = "Insurrection (Rev+Borers)"
```
Comment (NonQueueingMatt) [inline code/game/gamemodes/mixed/insurrection.dm]:
```suggestion
	name = "Insurrection (rev+borer)"
```

**Pull request merged**